doc/tutorials/running_ubun_as_user_vm.rst

```diff
@@ -9,14 +9,13 @@ Prerequisites
 This tutorial assumes you have already set up the ACRN Service VM on an
 Intel NUC Kit. If you have not, refer to the following instructions:
 
-- Install a `Clear Linux OS
-  <https://docs.01.org/clearlinux/latest/get-started/bare-metal-install-server.html>`_
+- Install a `Ubuntu 18.04 desktop ISO
+  <http://releases.ubuntu.com/18.04.3/ubuntu-18.04.3-desktop-amd64.iso?_ga=2.160010942.221344839.1566963570-491064742.1554370503>`_
   on your Intel NUC kit.
-- Follow the instructions at XXX to set up the
-  Service VM automatically on your Intel NUC kit. Follow steps 1 - 4.
+- Follow the instructions at `Build and Install ACRN on Ubuntu
```

Comment:
Same issue here as @gvancuts has mentioned.  We don't link to the generated HTML this way.  The bigger problem is you're linking to "latest" and not to the documentation for the current release (v2.3).  By using the :ref:`linkname` style of inter-document linking, the generated reference will be within that release's documentation set.  Very important.

Reply:
@dbkinder 
Thanks for comments!